LayoutTests/platform/mac-site-isolation/TestExpectations

```diff
@@ -5195,6 +5195,7 @@ http/wpt/html/semantics/scripting-1/the-script-element/module/module-meta-url-re
 http/wpt/html/semantics/scripting-1/the-script-element/module/module-meta-url-with-fragment.html [ Skip ]
 http/wpt/html/semantics/text-level-semantics/the-a-element/a-download-click-404.html [ Skip ]
 http/wpt/identity/identitycredentialscontainer-create-basics.https.html [ Skip ]
+http/tests/identity/allow-attribute.https.html [ Skip ]
```

Comment:
Do you know why this is failing?

Reply:
The iframe is not getting focus on EWS, but does so locally. 

I have an alternative solution (change the exception type to a SecurityError instead for unfocused docs). I sent a PR to update the spec.

Reply:
Actually, it seems I just forgot to call `.focus()` on the iframe 🙈 ... let's see if that fixes it.